packages/palette/package.json

```diff
@@ -71,7 +71,7 @@
     "@types/react-lazy-load-image-component": "1.3.0",
     "@types/react-test-renderer": "16.8.1",
     "@types/semver": "5.5.0",
-    "@types/styled-components": "4.0.2",
+    "@types/styled-components": "4.0.3",
```

Comment:
Looks like I didn't get this right; trying to find a version that has ThemeContext but doesn't catastrophically break all our typings (which some of the 4x releases do). Difficult to hunt down the specific version that `ThemeContext` was introduced with; could always `@ts-ignore` it.

Reply:
How are we still on this old styled-components version 🤯 Lets return to this first thing after we get this new version in.